src/Mobs/Path.h

```diff
@@ -0,0 +1,167 @@
+#pragma once
+
+#ifdef COMPILING_PATHFIND_DEBUGGER
+/* Note: the COMPILING_PATHFIND_DEBUGGER flag is used by Native/WiseOldMan95 to debug this class outside of MCServer.
+This preprocessor flag is never set when compiling MCServer. */
+#include "PathFinderIrrlicht_Head.h"
+#endif
+
+#include <vector>
+#include <queue>
```

Comment:
Including Globals.h in all files spawned hundreds of errors, mainly INSIDE globals.h,. Any idea?